app.js

```diff
@@ -1,8 +1,74 @@
+// Convert text to only alphabetic characters and spaces
+const removePunctuation = text => {
+    return text.replace(/[^ A-Za-z]/g,'');
+};
+
+// Convert string of text into an array of individual words
+const splitText = text => {
+    return text.split(' ');
+}
+
+// Remove banned words from an array of words
+const removeBannedWords = (words, banned) => {
+    const bannedWords = new Set(banned);
+    return words.filter(word => !bannedWords.has(word));
+}
+
+// Count the frequency of each word in an array of words
+const getWordCounts = words => {
+    const wordCounts = {};
+
+    for (let word of words) {
+        if (typeof wordCounts[word] === 'number') {
```

Comment:
I think there's a slightly shorter way to test wordCounts[word], possibly checking if it's undefined

Reply:
Your current way is the safer way but not necessary for this use, imo